packages/utilities/src/lib/findFilesRecursively.ts

```diff
@@ -0,0 +1,97 @@
+import type { PathLike } from 'node:fs';
```

Comment:
Hold up.

Since when do we have Node-only APIs in `@sapphire/utilities`?

cc: @favna since you opened the issue (sorry for not bringing this earlier, I didn't notice).

Reply:
Oh... right.. crap. I totally forgot when writing the issue. @imranbarbhuiya can you move this to its own package?